### ISSUE TITLE
Establish criteria for project inclusion

### ISSUE THREAD

**Issue by tfmorris** [okfn/okfn.github.com#408]
What are the criteria for being included the OKFN Labs [project page](http://okfnlabs.org/projects/#featured=true)? When I looked at the page today, top center was occupied by the "Puppycide" project. Inflammatory name aside, this prominence trivializes some of the real work that OKFN is trying to do.

Similarly the Bishop Ivo project appears to be a dumping ground of final format PDFs with no associated source files to allow them to be regenerated. Does that conform to OKFNs idea of best practices for publishing information?

If this answer is that currently there are no criteria, I'd recommend establishing some to stem the brand dilution.

**Comment by rgrp** [comment]
@tfmorris agreed. We are in process of generally trying to improve the classification of projects see #402 #407 etc.

We also want clearer criteria for project inclusion. I think best way is to finish the "vision" document so that we have a clear sense of priorities #409
